src/main/java/com/napier/sem/Queries/CountriesQueries.java

```diff
@@ -6,18 +6,19 @@
 import java.sql.ResultSet;
 import java.sql.Statement;
 import java.util.ArrayList;
+import java.util.List;
 
 public class CountriesQueries {
 
     //todo, add user input for continent, region, limit
     public static void getAllCountryReports(Connection con) {
         //get country data
-        ArrayList<Country> allCountries = getAllCountries(con);
-        ArrayList<Country> countriesInContinent = getCountriesInContinent("Europe", allCountries);
-        ArrayList<Country> countriesInRegion = getCountriesInRegion("Eastern Asia", allCountries);
-        ArrayList<Country> allCountriesLimited = getCountriesLimitedBy(3, allCountries);
-        ArrayList<Country> countriesInContinentLimited = getCountriesLimitedBy(3, countriesInContinent);
-        ArrayList<Country> countriesInRegionLimited = getCountriesLimitedBy(3, countriesInRegion);
+        List<Country> allCountries = getAllCountries(con);
```

Comment:
why are we changing this to `List` from `ArrayList`?

Reply:
I'm still returning an arraylist
Just abstracted so the implementation is only set in one place now (per list)